dhctl/pkg/app/ssh.go

```diff
@@ -154,6 +155,12 @@ func DefineBecomeFlags(cmd *kingpin.CmdClause) {
 		Envar(configEnvName("ASK_BECOME_PASS")).
 		Short('K').
 		BoolVar(&AskBecomePass)
+
+	if SSHUser == "root" {
+		_ = fmt.Errorf("flag --ssh-become-pass cannot be set for --ssh-user=root")
```

Comment:
1.`--ssh-become-pass` flag does't exist, do you mean `--ask-become-pass`
2.  error message will never be displayed, because you don't send created error to output
3.  does this condition work exactly when the user is specified via `--ssh-user=root` (without ENV USER=root)? please, check it.

Reply:
1. I'll fix it.
2. I'll fix it.
3. As I understand it, the `DefineSSHFlags` function processes arguments. If `--ssh-user` is specified, the `SSHUser` variable will receive the value from the argument. That is, when checking the `SSHUser `variable, I always get either the `USER` environment variable or the value from `--ssh-user`. Or does it not work that way?